webapp/client/src/components/FieldLabelScaffolding.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import ExampleFieldInput from "./ExampleFieldInput";
+import HelpButton from "./HelpButton";
+import HelpDialog from "./HelpDialog";
+import Details from "./Details";
+import OptionalHint from "./OptionalHint";
+
+export default function FieldLabelScaffolding({
+  render,
+  fieldId,
+  label,
+  details,
+}) {
+  // Only show label if there is text to show.
+  if (!label.text) {
+    return null;
+  }
+
+  const labelClassNames = classNames("text-input-field-label", {
+    "field-label": !label.exampleInput,
+    "field-label-example": label.exampleInput,
+  });
+  return (
+    <>
```

Comment:
This is shorthand for [`<React.Fragment>`](https://reactjs.org/docs/fragments.html). A React component must return exactly one element and this can be used to wrap several sibling elements in a parent. It does not generate any html.